fedmsg/core.py

```diff
@@ -369,7 +372,7 @@ def _create_poller(self, topic="", passive=False, **kw):
 
         # TODO -- the 'passive' here and the 'active' are ambiguous.  They
         # don't actually mean the same thing.  This should be resolved.
-        method = passive and 'bind' or 'connect'
+        method = (passive and 'bind') or 'connect'
```

Comment:
This change doesn't seem necessary, though I also don't think it is harmful. The order of operations already works this way.